pcsx2/SPU2/Mixer.cpp

```diff
@@ -585,37 +585,8 @@ static __forceinline StereoOut32 MixVoice(uint coreidx, uint voiceidx)
 
 	if (vc.ADSR.Phase > 0)
 	{
-		if (vc.Noise)
-			Value = GetNoiseValues(thiscore);
-		else
-		{
-			// Optimization : Forceinline'd Templated Dispatch Table.  Any halfwit compiler will
-			// turn this into a clever jump dispatch table (no call/rets, no compares, uber-efficient!)
-
-			switch (EmuConfig.SPU2.Interpolation)
-			{
-				case Pcsx2Config::SPU2Options::InterpolationMode::Nearest:
-					Value = GetVoiceValues<0>(thiscore, voiceidx);
-					break;
-				case Pcsx2Config::SPU2Options::InterpolationMode::Linear:
-					Value = GetVoiceValues<1>(thiscore, voiceidx);
-					break;
-				case Pcsx2Config::SPU2Options::InterpolationMode::Cubic:
-					Value = GetVoiceValues<2>(thiscore, voiceidx);
-					break;
-				case Pcsx2Config::SPU2Options::InterpolationMode::Hermite:
-					Value = GetVoiceValues<3>(thiscore, voiceidx);
-					break;
-				case Pcsx2Config::SPU2Options::InterpolationMode::CatmullRom:
-					Value = GetVoiceValues<4>(thiscore, voiceidx);
-					break;
-				case Pcsx2Config::SPU2Options::InterpolationMode::Gaussian:
-					Value = GetVoiceValues<5>(thiscore, voiceidx);
-					break;
-
-					jNO_DEFAULT;
-			}
-		}
+	
+		Value = GetVoiceValues<5>(thiscore, voiceidx);
```

Comment:
can you restore noise handling please?

Thanks

Reply:
probably should update GetVoiceValues too to not take a template number and just default to interpolating